### ISSUE TITLE
Please put SQUELCH control in ALL MODES! and ALL Bands!

### ISSUE THREAD

**Issue by N8SDR007** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#114]
Add Squelch control for ALL MODES> not just NFM and ALL Bands

Squelch is very handy for all modes, esp with noise

An all-mode squelch is a very handy feature to have!
1) Squelch is a very handy tool to reduce listening fatigue over long periods, no matter what mode in operation.

2) in a roundtable where all the signals are at a comparable level. It rivals a good FM situation with quietness

3) it's great for reducing distant static crashes (not strong one) but those in distance that are just annoying and have a weaker signal, if you squelch them, you can't hear them. 

I'm sure others can chime in for reasons that an all-mode squelch feature would make this a good feature. So those that read post below why you would also like an ALL-Mode Squelch.

**Comment by John-KC2QMA** [comment]
I agree, It was there in prior versions please put it back.

**Comment by s-car-go** [comment]
I agree as well. Squelch is very much used especially when I am monitoring for my buds to show up on the band.

**Comment by kingsqueak** [comment]
Yes would be useful, please add.

**Comment by ct1drb** [comment]
I believe squelch could be very useful in all modes and all bands and if in the past was working there is a strong reason to put it on again .

**Comment by IU4JNR** [comment]
I totally agree at all the point mentioned. The all mode squelch would be a great enhancement which I think it's also an easy fix.

**Comment by s58ww** [comment]
I agree at about all mentioned ... Squelch on all modes can be usefull.

**Comment by Hamuser** [comment]
yes please

**Comment by W1SPS** [comment]
You know, I didn't even realize the squelch wasn't there... Till I needed it. I have to agree, I'd like to see squelch available in all modes.

**Comment by N8SDR007** [comment]
So, you added it to AM and Sync AM, will it be implemented in ALL modes as many of us would really appreciate an ALL-mode Squelch option.  I understand programmer doesn't think itis needed, however programmer is not a radio operator that I am aware of, therefore he most likely doesn't see ALL benefits of this option.

**Comment by John-KC2QMA** [comment]
**ALL MODE SQUELCH** Please.

**Comment by SM0MDG** [comment]
Voice detected squelch in SSB please (not signal strength detection).

**Comment by N8SDR007** [comment]
Okay so you put it in SSB for the 1.0.2 beta- still not all modes? no CW or Digital mode squelch

**Comment by N8SDR007** [comment]
> Voice detected squelch in SSB please (not signal strength detection).

that be NICE :)

**Comment by rvolchkov** [comment]
Implemented in the latest 1.0.2.

**Comment by N8SDR007** [comment]
> Implemented in the latest 1.0.2.

as far as i know this is only in AM/FM and SSB

it is not in DigU or DigiL or CW. therefore not all mode- please reconsider and reopen this request

**Comment by N8SDR007** [comment]
Roman

 

SQL is only in AM/FM and SSB

 

It is not in all modes as the request was submitted

 

It is not in Digu or DigL nor is it in CW,

 

Please reopen the request and consider adding as suggested in the original request to ALL MODES

**Comment by rvolchkov** [comment]
We will not implement SQL in CW and DIGI

**Comment by N8SDR007** [comment]
Is there a reason why?

**Comment by rvolchkov** [comment]
It's not required for CW and DIGI. It's our opinion, thus we do not plan to implement SQL there.

**Comment by N8SDR007** [comment]
Yes your option

I see it otherwise, personally and other rigs have it.

So my option will be replacing it as main rig.



⁣Get BlueMail for Android ​

**Comment by W1SPS** [comment]
What about SSB?

**Comment by N8SDR007** [comment]
> What about SSB?

Rich

there answer is it Anit going to happen.

Apparently, they have no intention or even a clue how helpful it would be - which is why many of us have abandoned and moved away- oddly I forgot to kill off the email notifications for this, which is why I replied. as I didn't want to leave you hanging here with no answer- Rome said it above and that that time I made up my mine, those there which I still think is only a couple who actually operate a radio day to day don't see any need for all MODE including SSB squelch.  Other brands see this differently as they understand the benefits gained by allowing one too to control the breaking point of signal strength to open the front end of a receiver to allow for less noise, and a better overall experience, especially used in groups, RT or general ragchews. If that is what your seeking, then you like I did, find selling and moving to another brand will benefit you better. I just hate the GUI I now have to look at, but at least all the options are there, and they work! no smoke n mirrors
